dbms/src/Storages/Transaction/ProxyFFIStatusService.cpp

```diff
@@ -81,7 +81,10 @@ HttpRequestRes HandleHttpRequestSyncStatus(
                 }
             }
             count = region_list.size();
```

Comment:
Better rename "count" -> "ready_region_count", and use "=" instead of ":" in line 78